.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 12
```

Comment:
Do you think it would be more sustainable to use lts/* here and lts-slim in the docker file to make it simple to stay on lts?

Reply:
I'm not sure that it's a good idea to automatically upgrade major versions of node. I do have tests that would need to pass before it's deployed. Still...🤔

I think that it's probably best to be deliberate with major version upgrades. But I'm open to alternative opinions.

Reply:
I can respect that decision. I was influenced by cloud functions etc where pinned runtimes are eventually deprecated.